eclipse-scout-core/src/form/fields/datefield/DateField.js

```diff
@@ -1385,10 +1385,16 @@ export default class DateField extends ValueField {
     this.popup = this.createDatePopup();
     this.popup.open();
     this.$dateField.addClass('focused');
-    this.popup.on('remove', event => {
-      this._onPopupRemove(event);
-      this.popup = null;
+    // After picking a date the popup gets closed and destroyed. The method Widget#destroy triggers a 'remove' event
+    // and then tries to remove all children from the owner (i.e. the date field). Do not register the following popup
+    // listener on the 'remove' event, because it will destroy the date field in touch mode! That will lead to an error,
+    // when the popup tries to remove the children from the no longer existing owner.
+    this.popup.on('close', event => {
```

Comment:
Please use destroy instead of close, because close can be prevented and destroy will be triggered at the very end.
Please also use one instead of on. Although it should not be possible that it will be executed more than once, it is semantically more correct.